### PR TITLE
Added campaign return link and changed button text

### LINKED ISSUE #38: Campaign Stage Title change
image

Format should be:
(Campaign Name) - Stage (stage number)

Also need a link to the main campaign page.

Instead of Commit Funds say "Support This Project"

### PR DESCRIPTION
**Pull request by UsamaSaleem14**
image

### CODE REVIEW

**Comment by TBomberman** [comment]
Can you make this a back arrow? (like in gs)

**Comment by UsamaSaleem14** [comment]
> Can you make this a back arrow? (like in gs)

Added an arrow instead of text link

image

File: src/hooks/useSingleStageHandler.ts [hunk 1]
```diff
@@ -816,5 +816,6 @@ export const useSingleStageHandler = () => {
     commitERCAmount,
     tokensCommittedEth,
     maxVoteWeight,
+    id,
```
Comment (TBomberman) [inline src/hooks/useSingleStageHandler.ts]: rename to campaignId

Comment (TBomberman) [reply src/hooks/useSingleStageHandler.ts]: done

**Review by TBomberman — APPROVED** [review]

**Pull request merged**